src/porter/porter.objectmodel.ts

```diff
@@ -46,3 +46,36 @@ export function isPath(s: CredentialSource): s is PathCredentialSource {
 export function isCommand(s: CredentialSource): s is ShellCommandCredentialSource {
     return (s as any).command;
 }
+
+export interface Installation {
+    readonly Name: string;
+    readonly Created: string;
+    readonly Modified: string;
+    readonly Action: string;
+    readonly Status: string;
+    readonly History: ReadonlyArray<InstallationHistoryEntry>;  // only has latest in practice
+}
+
+export interface InstallationDetail {
+    readonly Name: string;
+    readonly Created: string;
+    readonly Modified: string;
+    readonly Action: string;
+    readonly Status: string;
+    readonly Outputs: ReadonlyArray<InstallationOutput>;
+    readonly History: ReadonlyArray<InstallationHistoryEntry>;
+}
+
+export interface InstallationHistoryEntry {
+    readonly ClaimID: string;
+    readonly Action: string;
+    readonly Timestamp: string;
+    readonly Status: string;
+    readonly HasLogs: boolean;
```

Comment:
I don't think the context menu option for View Logs is using this field. I wanted you to know it exists if we want to disable that menu option when the logs aren't available.

Reply:
Okay, I can use this to set the context value on the tree node.  I think I can only hide rather than disable menu options though.